radio/src/gui/128x64/model_setup.cpp

```diff
@@ -85,8 +85,10 @@ enum MenuModelSetupItems {
   ITEM_MODEL_SETUP_CHECKLIST_DISPLAY,
   ITEM_MODEL_SETUP_THROTTLE_WARNING,
   ITEM_MODEL_SETUP_SWITCHES_WARNING1,
+  #if defined(PCBTARANIS)
   ITEM_MODEL_SETUP_SWITCHES_WARNING2,
   ITEM_MODEL_SETUP_POTS_WARNING,
```

Comment:
Er... the 9XR-Pro has pots also... think you roped in one line to many there ;)

Reply:
Yes, it has pots, but for some reason, the pot warning was only ever working on PCBTARANIS. To get the pot warnings working on 9XR-Pro requires some changes in other parts of the code too. Since it seems the feature was originally intended only for PCBTARANIS, I figured it would be safer to just put it back the way it was.

Reply:
Indeed... a non-broken menu is always a good start. I'll have to dig mine out and get it working again... it's not been used for a few years now due to the various gremlins that have been creeping in for the 9X and 9XRPRO targets... might be time to finally try and fix some of them! :)